addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6208,7 +6208,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#13435"
-msgid "Enable HQ Scalers for scalings above"
+msgid "Enable HQ scalers for scaling above"
```

Comment:
why singular for scaling? I'm not a native speaker but it feels wrong.  @jjd-uk ?